references/classification/utils.py

```diff
@@ -344,7 +344,7 @@ def store_model_weights(model, checkpoint_path, checkpoint_key="model", strict=T
 
         # Quantized Classification
         model = M.quantization.mobilenet_v3_large(pretrained=False, quantize=False)
-        model.fuse_model()
+        model.fuse_model(is_qat=True)
```

Comment:
would you like to use fuse_model and fuse_model_qat or the flag?

Reply:
Thanks for the comment. Apologies, I'm not sure I follow. Could you clarify?